### ISSUE TITLE
mask_rcnn_facedetection_0001.h5?

### ISSUE THREAD

**Issue by haoopan** [stonePJ/Face-Detection-Base-on-Mask-R-CNN#1]

**Comment by stonePJ** [comment]
`model.train(dataset_train, dataset_val, learning_rate=config.LEARNING_RATE, epochs=1, layers='heads')`
Increase epochs could be better!
The best way is to use this modle [mask_rcnn_coco.h5](https://github.com/matterport/Mask_RCNN/releases/download/v2.0/mask_rcnn_coco.h5)
Train wtih the model of coco, the result will be better!
Review the code:
```
if args.command == "train":
        model = modellib.MaskRCNN(mode="training", config=config, model_dir='F:/model_dir')
        model.load_weights(model.find_last()[1], by_name=True)
        train(model)
```

**Comment by nyck33** [comment]
@f1ashine Do you mean we can use the mask_rcnn_coco.h5 weights and then just train the heads on our new dataset?
